### PR TITLE
Css fixes

### PR DESCRIPTION
**Pull request by birdbrained**

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -867,164 +965,3 @@ span.sortable-bars:hover {
...
-.input-box{
-	text-align: center;
-	margin: auto;
-}
-
-
-.form-holder{
-	max-width: 800px;
-	margin: 0 auto
-}
```
Comment (birdbrained) [inline app/styles/app.scss]: moved

File: app/styles/components/contributor-list.scss [hunk 2]
```diff
@@ -1,4 +1,12 @@
-.contributor-list-container {
```
Comment (birdbrained) [inline app/styles/components/contributor-list.scss]: avoid "container"

File: app/styles/components/landing-paragraph.scss [hunk 3]
```diff
@@ -1,12 +1,11 @@
+@import "../variables";
+
 .landing-paragraph-container {
-  p {
-    text-align: justify;
-    text-justify: inter-word;
-    word-break: break-all;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
```
Comment (birdbrained) [inline app/styles/components/landing-paragraph.scss]: may not need this

File: app/styles/components/landing-sponsors.scss [hunk 4]
```diff
@@ -2,8 +2,35 @@
   height: 80px;
 }
 
-.sponsors-logo {
-  max-width: 150px;
-  max-height: 60px;
-  mix-blend-mode: luminosity;
-}
```
Comment (birdbrained) [inline app/styles/components/landing-sponsors.scss]: now images are backgrounds

File: app/styles/components/meetings-schedule.scss [hunk 5]
```diff
@@ -53,4 +110,160 @@
...
+	padding: 16px;
+	z-index: 10;
+}
+
+select{
+	width: 100%;
+	height: 32px;
+	background-color: white;
+	border: 1px solid rgba(0,0,0,0.125);
+	margin-bottom: 16px;
```
Comment (birdbrained) [inline app/styles/components/meetings-schedule.scss]: these could interfere with regular styles. need to be namespaced; though these have only been moved from app.css- theyre styles that were added along with the filter bar for the schedule. Should leave for now - this pr is already big. 

In the future need to be careful when ^c ^v from the internet....

File: app/styles/mixins.scss [hunk 6]
```diff
@@ -4,7 +4,19 @@
 
 @mixin panel {
     background: #ffffff;
-    padding: 20px;
     margin: 15px 15px;
+    padding: 20px;
 
 }
+
+@mixin narrow-panel {
```
Comment (birdbrained) [inline app/styles/mixins.scss]: might not need "narrow-panel"

File: app/templates/collections/collection/edit.hbs [hunk 7]
```diff
@@ -1,59 +1,31 @@
-<div class="coll-add-layer p-xl">
-    <div class="container">
-        {{# if editMode}}
+{{# if editMode}}
```
Comment (birdbrained) [inline app/templates/collections/collection/edit.hbs]: I dono what happened to this file. Need to chat about the merge conflicts here. edit looks really poor rn.

File: app/templates/create.hbs [hunk 8]
```diff
@@ -1,62 +1,58 @@
-<div class="hero">
-    <h2>Create a collection</h2>
-</div>
-<div class="form-holder">
+<form>
+    <h1>Create a collection</h1>
 
-    <section class="form-section">
-        <h4>Add title </h4>
+    <fieldset>
```
Comment (birdbrained) [inline app/templates/create.hbs]: A filedset is a grouping element for related fields. perfect for "section"

File: package.json [hunk 9]
```diff
@@ -72,7 +72,7 @@
     "loader.js": "^4.0.1",
     "lodash": "^4.17.4",
     "lodash.omit": "^4.5.0",
-    "pagination-pager": "2.4.2",
+    "pagination-pager": "^3.1.0",
```
Comment (birdbrained) [inline package.json]: why not...

File: app/components/contributor-list/template.hbs [hunk 10]
```diff
@@ -1,41 +1,45 @@
-<div id="{{get-section-id layout.title}}" class="contributor-list-container p-t-xl p-b-xl"
-     style="{{containerStyle}}">
-    <div class="container">
-        {{#if layout.title}}
-            <div class="row text-center section-title">
-                <h2>
-                    {{layout.title}}
-                </h2>
+{{#if layout.title}}
```
Comment (birdbrained) [inline app/components/contributor-list/template.hbs]: this can be simplified more, no need for so many conditional if we use a computed on the model.... pr is too big as is, so will make a separate pr. This component should appear the same as it did before as it is now though.

File: app/components/meeting-schedule/template.hbs [hunk 11]
```diff
@@ -1,82 +1,79 @@
...
+            {{/each}}
+
         </div>
-    </div>
+
+    {{/each}}
+</div>
+
+<div class="item-detail">
+    <h3>{{get (await selectedItem) 'title'}}<br>
```
Comment (birdbrained) [inline app/components/meeting-schedule/template.hbs]: TODO: avoid await in templates

File: app/components/file-uploader/template.hbs [hunk 12]
```diff
@@ -1,36 +1,3 @@
-<style>
```
Comment (Rytiggy) [inline app/components/file-uploader/template.hbs]: Good Call

**Comment by Rytiggy** [comment]
Looks good

File: app/components/dataset-file-grid/component.js [hunk 13]
```diff
@@ -2,9 +2,14 @@ import Ember from 'ember';
 import pagedArray from 'ember-cli-pagination/computed/paged-array';
 
 export default Ember.Component.extend({
+
     store: Ember.inject.service(),
     session: Ember.inject.service(),
-    
+    router: Ember.inject.service('-routing'),
+
+    classNames: ["item-details"],
```
Comment (cameronblandford) [inline app/components/dataset-file-grid/component.js]: these class names for each section aren't standardized. This one is "item-details", another one is "items", another one is "landing-paragraph-container". They should all be named according to the same convention

**Comment by cameronblandford** [comment]
Looks good! Merging
